cleverpush/src/main/java/com/cleverpush/service/CleverPushGeofenceTransitionsIntentService.java

```diff
@@ -22,51 +25,51 @@
 import java.util.List;
 
 public class CleverPushGeofenceTransitionsIntentService extends IntentService {
-	protected static final String TAG = "CPGeofenceTransitionsIS";
+    protected static final String TAG = "CPGeofenceTransitionsIS";
 
-	public CleverPushGeofenceTransitionsIntentService() {
-		super(TAG);  // use TAG to name the IntentService worker thread
-	}
+    public CleverPushGeofenceTransitionsIntentService() {
+        super(TAG);  // use TAG to name the IntentService worker thread
+    }
 
-	@Override
-	protected void onHandleIntent(Intent intent) {
-		GeofencingEvent event = GeofencingEvent.fromIntent(intent);
-		if (event.hasError()) {
-			Logger.e(TAG, "GeofencingEvent Error: " + event.getErrorCode());
-			return;
-		}
+    @Override
+    protected void onHandleIntent(Intent intent) {
 
-		SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(this);
+        GeofencingEvent event = GeofencingEvent.fromIntent(intent);
+        if (event.hasError()) {
+            Logger.e(TAG, "GeofencingEvent Error: " + event.getErrorCode());
+            return;
+        }
 
-		String channelId = sharedPreferences.getString(CleverPushPreferences.CHANNEL_ID, null);
-		String subscriptionId = sharedPreferences.getString(CleverPushPreferences.SUBSCRIPTION_ID, null);
-		String transitionState = event.getGeofenceTransition() == Geofence.GEOFENCE_TRANSITION_ENTER ? "enter": "exit";
+        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(this);
 
-		Logger.d(TAG, "Geofence Transition Details: " + getGeofenceTransitionDetails(event) + " " + transitionState + " subscription: " + subscriptionId + " channel " + channelId);
+        String channelId = sharedPreferences.getString(CleverPushPreferences.CHANNEL_ID, null);
+        String subscriptionId = sharedPreferences.getString(CleverPushPreferences.SUBSCRIPTION_ID, null);
+        String transitionState = event.getGeofenceTransition() == Geofence.GEOFENCE_TRANSITION_ENTER ? GEOFENCE_ENTER_STATE : GEOFENCE_EXIT_STATE;
 
-		if (channelId != null && subscriptionId != null) {
-			for (Geofence geofence : event.getTriggeringGeofences()) {
-				JSONObject jsonBody = new JSONObject();
-				try {
-					jsonBody.put("geoFenceId", geofence.getRequestId());
-					jsonBody.put("channelId", channelId);
-					jsonBody.put("subscriptionId", subscriptionId);
-					jsonBody.put("state", transitionState);
+        if (channelId != null && subscriptionId != null) {
+            for (Geofence geofence : event.getTriggeringGeofences()) {
+                JSONObject jsonBody = new JSONObject();
+                try {
+                    jsonBody.put("geoFenceId", geofence.getRequestId());
+                    jsonBody.put("geoFenceId", "geofence.getRequestId()");
```

Comment:
copy paste mistake?

Reply:
+ resolve conflicts

Reply:
Yes sorry it's copy-paste mistake 
and now all the conflicts are resolved